.github/workflows/docusaurus.yml

```
@@ -0,0 +1,24 @@
+name: docusaurus
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build:
+    name: Build Docusaurus
```

to align with our existing job names
```suggestion
    name: build
```